src/components/idleMembers/card/index.tsx

```diff
@@ -1,17 +1,22 @@
 import { FC } from 'react';
 import classNames from '@/components/idleMembers/card/card.module.scss';
 
+const dummyProfile = 'dummyProfile.png';
+const url = process.env.NEXT_PUBLIC_GITHUB_IMAGE_URL;
```

Comment:
```suggestion
const imgUrl = process.env.NEXT_PUBLIC_GITHUB_IMAGE_URL;
```
`url` is very vague here, we can rename it to something more meaningful. What say?